src/data/set/intervals/interval.lean

```diff
@@ -0,0 +1,137 @@
+/-
+Copyright (c) 2020 Zhouhang Zhou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Zhouhang Zhou
+-/
+
+import data.set.intervals
+
+/-!
+# Intervals
+
+`[a, b]` is the set of elements lying between `a` and `b`, with both endpoints included.
+
+`Icc a b` requires the assumption `a ≤ b` to be meaningful, which is sometimes inconvenient. This
+version of interval is always the set of things lying between `a` and `b`, regardless of the
+relative order of `a` and `b`.
+-/
```

Comment:
Could you add a comment on your notation `[a, b]`, and the locale one should open to make it available?